### PR TITLE
Hdrp/update frame settings image

### PR DESCRIPTION
**Pull request by JordanL8**
### Purpose of this PR
Updated frame settings image and added the **Transparents Write Velocity** checkbox.

### CODE REVIEW

**Comment by JordanL8** [comment]
Not sure if the Transparents Write Velocity feature is for 2019.1 or .2, so I will leave whether or not it needs backporting up to you!

File: com.unity.render-pipelines.high-definition/Documentation~/Frame-Settings.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ These settings determine the method that the Cameras and Reflection Probes using
...
 | **Transparent Prepass**           | Enable this checkbox to make HDRP perform a Transparent Prepass. Enabling this feature causes HDRP to add polygons from transparent Materials to the depth buffer to improve sorting. |
 | **Transparent Postpass**          | Enable this checkbox to make HDRP perform a Transparent Postpass. Enabling this feature causes HDRP to add polygons to the depth buffer that postprocessing uses. |
+| **Transparents Write Velocity**   | Enable this checkbox to allow HDRP to write the velocity of transparent GameObjects to the velocity buffer. To make HDRP write transparent GameObjects to the velocity buffer, you must also enable the **Transparent Writes Velocity** checkbox on each transparent Material. Enabling this feature means that effects, such as motion blue, affect transparent GameObjects.  This is useful for alpha tested objects like hair. |
```
Comment (sebastienlagarde) [inline com.unity.render-pipelines.high-definition/Documentation~/Frame-Settings.md]: motion blue => motion blur
This is useful for alpha tested objects like hair. => This is useful for alpha blended objects like hair.

**Comment by JordanL8** [comment]
Made requested changes. Also, should I be targeting master or hdrp/staging here?

**Comment by sebastienlagarde** [comment]
Master in this case as I will bakport this change.

**Pull request merged**